pygments/lexers/ptx.py

```diff
@@ -0,0 +1,109 @@
+import re
+from pygments.lexer import RegexLexer, include, words
+from pygments.token import Comment, Keyword, Name, String, Number, Punctuation, \
+Whitespace, Operator
+
+__all__ = ["PtxLexer"]
+
+
+class PtxLexer(RegexLexer):
+    """
+    For NVIDIA `PTX <https://docs.nvidia.com/cuda/parallel-thread-execution/>`_
+    source.
+
+    .. versionadded:: 2.15
+    """
+    name = 'PTX'
+    filenames = ['*.ptx']
+    aliases = ['ptx']
+
```

Comment:
Please add `url = "https://docs.nvidia.com/cuda/parallel-thread-execution/"` here which will allow to link to the site from various places.